task.js

```diff
@@ -28,7 +28,17 @@ class Task {
     this.mapper = mapper || function() { return ''; };
     this.translations = [];
   }
+  static async logError(title,error){
+    var today = new Date();
+    var date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
+    var time = today.getHours() + ":" + today.getMinutes() + ":" + today.getSeconds();
+    var dateTime = date+' '+time;
+    const otfl = `${OUTPUT_DIRECTORY}/errors.txt`;
+    //adding time at the beginig of the error and \n at the end?
+    let er = dateTime+ ', Task title: ' + title + ': ' + error +'\n';
```

Comment:
No need to add  a timestamp an task title on each line. `\n` at the end is fine, unless `fs.appendFile` automatically puts the  message on a new line?